db/migrate/20190418062823_create_api_keys.rb

```diff
@@ -0,0 +1,19 @@
+class CreateApiKeys < ActiveRecord::Migration[5.2]
+  def change
+    create_table :api_keys do |t|
+      t.references :user, foreign_key: true, index: true, null: false
+      t.string     :name, null: false
+      t.string     :hashed_key, index: true, null: false
```

Comment:
Consider adding a unique index to `hashed_key` to prevent duplicates (admittedly unlikely but best avoided).

Reply:
Done. Thanks :+1: